code/modules/mob/living/say.dm

```diff
@@ -300,13 +300,40 @@ GLOBAL_LIST_INIT(special_radio_keys, list(
 		eavesdropping = stars(message)
 		eavesrendered = compose_message(src, message_language, eavesdropping, , spans, message_mods)
 
+	var/voice = MASC1
+	if(ishuman(src))
+		var/mob/living/carbon/human/H = src
+		if(GLOB.tts_enum[voice])
+			voice = H.tts
+		else
+			voice = MASC1
```

Comment:
Voice is still not sanitizied:
- tts_enum can be controlled by admins
- H.tts can be controlled by admins

Reply:
resolved in [239c0e0](https://github.com/yogstation13/Yogstation/pull/14775/commits/239c0e065fc83f90a95cc36c4edce8d78ba66c3f)